spec/spec_helper.rb

```diff
@@ -1,6 +1,6 @@
-# require 'codeclimate_env.rb'
-# require 'codeclimate-test-reporter'
-# CodeClimate::TestReporter.start
+require 'codeclimate_env.rb'
+require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.